test_conformance/math_brute_force/macro_unary.c

```diff
@@ -564,6 +564,7 @@ static cl_int TestFloat( cl_uint job_id, cl_uint thread_id, void *data )
     }
 
 exit:
+    cl_int ret = error;
```

Comment:
This caused a build error in cl12_trunk. I'll push a fix to declare it at the top instead.